autoshop/core/forms.py

```diff
@@ -31,3 +33,19 @@ class SigninForm(forms.Form):
     username = forms.CharField(max_length=100)
     password = forms.CharField(widget=forms.PasswordInput, required=True)
 
+
+class EditProfileForm(ModelForm):
+    """
+    Edit Profile form.
+
+    fields:
+        all fields from Profile model
```

Comment:
remove it as it is not providing any useful information.  we can clearly see that its a `Profile` model form